### PR TITLE
migrate to enhanced enum

### PR DESCRIPTION
**Pull request by utamori**
Migrate to Dart2.17 enhanced enum to increase readability

### CODE REVIEW

File: adaptive_breakpoints/lib/src/adaptive_breakpoints.dart [hunk 1]
```diff
@@ -13,19 +13,23 @@ import 'package:flutter/material.dart';
...
-    required this.widthRangeValues,
-    required this.heightLandscapeRangeValues,
-    required this.heightPortraitRangeValues,
-  });
 
+enum AdaptiveWindowType {
   /// Name based on the [AdaptiveWindowType].
   ///
   /// Can be: xsmall, small, medium, large or xlarge
-  final String name;
```
Comment (guidezpl) [inline adaptive_breakpoints/lib/src/adaptive_breakpoints.dart:28]: Please add trailing commas to make this easier to read

Comment (utamori) [reply adaptive_breakpoints/lib/src/adaptive_breakpoints.dart:28]: Thanks. Fixed.

File: adaptive_breakpoints/lib/src/adaptive_breakpoints.dart [hunk 2]
```diff
@@ -18,18 +18,46 @@ enum AdaptiveWindowType {
...
+    RangeValues(1440, 1919),
+    RangeValues(960, 1279),
+    RangeValues(1920, double.infinity),
+  ),
+  xlarge(
+    4,
+    RangeValues(
+      1920,
+      double.infinity,
+    ),
```
Comment (guidezpl) [inline adaptive_breakpoints/lib/src/adaptive_breakpoints.dart]: nit: 
```suggestion
    RangeValues(1920, double.infinity),
```

Comment (utamori) [reply adaptive_breakpoints/lib/src/adaptive_breakpoints.dart]: Thanks.

**Pull request closed without merging**